tasks/abs.rb

```diff
@@ -68,7 +68,7 @@ def provision(platform, inventory_location, vars)
 
   raise 'Timeout: unable to get a 200 response in 10 minutes' if reply.code != '200'
 
-  inventory_full_path = File.join(inventory_location, 'inventory.yaml')
+  inventory_full_path = File.join(inventory_location, '/spec/fixtures/litmus_inventory.yaml')
```

Comment:
I initially wanted to call out the extra `/` at the start there, but it seems like ruby already does what I feared would happen, even without the `/`:

```
david@zion:~$ pry
[1] pry(main)> File.join("", "/foo")
=> "/foo"
[2] pry(main)> File.join("", "foo")
=> "/foo"
[3] pry(main)> 
```